tests/v2/actions/charts/payloads.go

```diff
@@ -111,3 +111,27 @@ func newChartUpgrade(chartName, releaseName, version, clusterID, clusterName, ur
 
 	return &chartUpgrade
 }
+
+// newExtensionsInstallAction is a private constructor that creates a chart install with given chart values that can be used for chart install action.
+func newExtensionsInstall(name, version string, chartValues map[string]interface{}) *types.ChartInstall {
+	chartInstall := types.ChartInstall{
+		ChartName:   name,
+		ReleaseName: name,
+		Version:     version,
+		Values:      nil,
+	}
+
+	for k, v := range chartValues {
+		chartInstall.Values[k] = v
+	}
+
+	return &chartInstall
+}
+
+// newExtensionsInstallAction is a private constructor that creates a payload for extensions install action chartInstalls and extensions namespace.
+func newExtensionsInstallAction(namespace string, chartInstalls []types.ChartInstall) *types.ChartInstallAction {
```

Comment:
Since it's used in only one place and to avoid abstractions, would it be possible to remove this function?

Reply:
I'm following the chart structure that we have currently have similar to this https://github.com/rancher/rancher/blob/main/tests/v2/actions/charts/payloads.go#L36.. 
but sure I can remove it as its not  much of a value add.